tests/js/spec/views/performance/metricsSwitch.spec.tsx

```diff
@@ -8,11 +8,13 @@ import {
   useMetricsSwitch,
 } from 'sentry/views/performance/metricsSwitch';
 
+const handleSwitch = jest.fn();
+
 function TestComponent() {
   const {isMetricsData} = useMetricsSwitch();
   return (
     <Fragment>
-      <MetricsSwitch />
+      <MetricsSwitch onSwitch={handleSwitch} />
```

Comment:
Can we add an assertion in `it('toggles between transactions and metrics')` test to make sure that handleSwitch mock gets called?